ui-lib/src/main/java/co/electriccoin/zcash/ui/screen/onboarding/view/OnboardingView.kt

```diff
@@ -98,21 +126,46 @@ private fun OnboardingTopAppBar(onboardingState: OnboardingState) {
             if (IS_NAVIGATION_IN_APP_BAR && currentStage.hasNext()) {
                 NavigationButton(onboardingState::goToEnd, stringResource(R.string.onboarding_skip))
             }
+
+            if (isDebugMenuEnabled) {
+                DebugMenu(onFixtureWallet)
+            }
         }
     )
 }
 
+@Composable
+private fun DebugMenu(onFixtureWallet: () -> Unit) {
+    var expanded by rememberSaveable { mutableStateOf(false) }
+    IconButton(onClick = { expanded = true }) {
+        Icon(Icons.Default.MoreVert, contentDescription = null)
+    }
+
+    DropdownMenu(
+        expanded = expanded,
+        onDismissRequest = { expanded = false }
+    ) {
+        DropdownMenuItem(
+            text = { Text("Import wallet with fixture seed phrase") },
+            onClick = onFixtureWallet
+        )
+    }
+}
+
 /**
  * @param onImportWallet Callback when the user decides to import an existing wallet.
  * @param onCreateWallet Callback when the user decides to create a new wallet.
  */
 @Composable
 fun OnboardingMainContent(
+    paddingValues: PaddingValues,
     onboardingState: OnboardingState,
     onImportWallet: () -> Unit,
     onCreateWallet: () -> Unit
 ) {
-    Column {
+    Column(Modifier
+        .padding(top = paddingValues.calculateTopPadding())
```

Comment:
Here I used the same pattern for padding as in my current Compose Material3 PR to have it compatible once merged together.